docs/README.md

```diff
@@ -1,12 +1,6 @@
 ## Table of Contents
 
 * [Read Me](/README.md)
-* [Introduction](/docs/introduction/README.md)
```

Comment:
we are maintaining these links here so they appear on https://travix-international.github.io/frint.

we go though the docs in steps.